display_menu.rb

```diff
@@ -0,0 +1,83 @@
+require_relative './exit_app'
+
+def display_menu(app)
+  puts "\nWelcome to our Catalog!"
+  puts "\nPlease choose a task by entering a number:"
+  puts " 1 - For listing items\n 2 - For adding items\n 3 - Exit App"
+  choice = gets.chomp
+  case choice
+  when '1'
+    for_listing(app)
+  when '2'
+    for_adding(app)
+  when '3'
+    exit_app(app)
+  else
+    puts 'Invalid selection'
+  end
+end
+
+def for_listing(app)
+  puts "\nWelcome to our Catalog!"
+  puts "\nPlease select a number for listing items"
+  puts " 1 - List all books\n 2 - List all music albums\n 3 - List all games\n 4 - For next menu\n 5 - Back to main\n 6 - Exit App" # rubocop:disable Layout/LineLength
+  choice = gets.chomp
+  case choice
+  when '1'
+    app.book_display
+  when '2'
+    app.music_display
+  when '3'
+    app.game_display
+  when '4'
+    display_next(app)
+  when '5'
+    display_menu(app)
+  when '6'
+    exit_app(app)
+  else
+    puts 'Invalid selection'
```

Comment:
Nice job displaying an invalid selection! For improved readability, please add a delay after displaying this message.

Reply:
@masangana I have this method in my last project that was used to create a delay in a message.
 
![image](https://user-images.githubusercontent.com/93630700/188508429-a6e617ba-262f-4d68-9890-b7cdc4f18f56.png)